src/common/colorspaces_inline_conversions.h

```diff
@@ -421,6 +422,125 @@ static inline float dt_camera_rgb_luminance(const float *const rgb)
   return (rgb[0] * 0.2225045f + rgb[1] * 0.7168786f + rgb[2] * 0.0606169f);
 }
 
```

Comment:
Minor : it's good practice to include source in comment here : https://www.osapublishing.org/oe/fulltext.cfm?uri=oe-25-13-15131&id=368272

Reply:
Sorry, added a comment with the information

Reply:
cool, thanks